### DISCUSSION TITLE
zigbee2mqtt.service won't activate in systemctl

### DISCUSSION THREAD

**Discussion by bobino-pepino** [Koenkk/zigbee2mqtt#18368] [Devices]
I'm trying run a zigbee2mqtt service using systemctl, but when I start it, it stays in the _activating_ state and never moves onto to the _active_ state.

It used to work fine before I re-flashed my Raspberry Pi 4b with the latest OS (Bullseye).

The program works fine when I run it from the command line.

If I run <systemctl list-units --type=service> i get the following response:

'pi@AbrigoHub:~/shared/bridge $ systemctl list-units --type=service

  UNIT                                                         LOAD   ACTIVE     SUB          DESCRIPTION
  alsa-restore.service                                  loaded active     exited       Save/Restore Sound Card State
  avahi-daemon.service                             loaded active     running      Avahi mDNS/DNS-SD Stack
  bluetooth.service                                    loaded active     running      Bluetooth service
  .
  .
  .
  vncserver-x11-serviced.service                              loaded active     running      VNC Server in Service Mode daem>
  wpa_supplicant.service                                      loaded active     running      WPA supplicant
  zigbee2mqtt.service                                         loaded activating auto-restart zigbee2mqtt
'

**Comment by Nardol** [comment]
Could you post the content of your SystemD unit file and the output of `systemctl status zigbee2mqtt.service` please?
Or the content of `journalctl --unit zigbee2mqtt.service -f` if there are too long lines.
It looks like the service restarts in loop.
